Packs/CimTrak-SystemIntegrityAssurance/IncidentTypes/incidenttype-CimTrak_Incident.json

```diff
@@ -0,0 +1,28 @@
+{
+    "autorun": false,
+    "closureScript": "GenerateInvestigationSummaryReport",
+    "color": "#01A2EC",
+    "days": 0,
+    "daysR": 0,
+    "default": false,
+    "detached": false,
+    "disabled": false,
+    "extractSettings": {
+        "fieldCliNameToExtractSettings": {},
+        "mode": "Specific"
+    },
+    "hours": 0,
+    "hoursR": 0,
+    "id": "CimTrak Incident",
+    "locked": false,
+    "name": "CimTrak Incident",
+    "onChangeRepAlg": 0,
+    "playbookId": "c526be83-711d-4212-8b5a-e618470f1958",
```

Comment:
@kigerjoel Please change to the playbook id.

Reply:
Fixed